src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -10287,85 +10287,85 @@ namespace SurfaceGeometry {
                 ErrorsFound = true;
             }
 
+            enum Month
```

Comment:
`enum class`? Or do you not want to `static_cast<int>` this everywhere.

Reply:
That's fine. One thing I would do, however, is move this stuff out of `SurfaceGeometry`. This seems more like a `Schedule` or `General` or `Utility` thing. You shouldn't have to `#include "SurfaceGeometry.hh"` if you want to use `enum Month`.

Reply:
The fact that `enum class` do not support subtyping is annoying. Why can't an `enum class` be a subtype of `int` or `char` or even `bool`? Why can't one `enum class` be a subtype, i.e., an extension, of another `enum class`? Not a super well thought-out or executed feature IMO. Or maybe I am missing some booby-trap.

Reply:
I think given that this is a plain enum, not a scoped enum class, I'm not going to move it to General _for right now_.  It would be good to take a brief, but intelligent, grep/scan through the code looking for usages of months, days in month, other stuff, and collect them all into a single section of enum/enum class with accompanying std::arrays.  I'd like to avoid doing that here if that's alright with you.